Source/carthage/Build.swift

```diff
@@ -213,32 +213,10 @@ public func buildableSDKs(sdks: [SDK], _ scheme: String, _ configuration: String
 			return SignalProducer(values: sdks).map { ($0, identities) }
 		}
 		.flatMap(.Concat) { (sdk: SDK, signingIdentities: [CodeSigningIdentity]) -> SignalProducer<SDK, CarthageError> in
```

Comment:
I think we can replace these with just
```swift
return SignalProducer(values: sdks)
    .flaMap(.Concat) { sdk: SDK -> SignalProducer<SDK, CarthageError> in 
        identityCheckArgs.sdk = sdk
 	return BuildSettings.loadWithArguments(identityCheckArgs)
    }
    .take(1)
```

I might have missed something since I coded this in Safari :sweat_smile: